guildwars2/guild/sync.py

```diff
@@ -175,11 +179,42 @@ def check(m):
             return
         await self.bot.database.set_guild(ctx.guild, {"sync.on": on_off}, self)
         if on_off:
-            msg = ("Synchronization enabled.")
+            msg = "Synchronization enabled."
         else:
-            msg = ("Synchronization disabled.")
+            msg = "Synchronization disabled."
         await ctx.send(msg)
 
+    @guildsync.command(name="purge")
+    async def sync_purge(self, ctx, on_off: bool):
+        """Toggles removal of members that are not in the attached guild sync guild. Members without any other role
```

Comment:
First line (short description) should be shorter - its the one used in help command